### PR TITLE
update privilege management document

### PR DESCRIPTION
**Pull request by imtbkcat**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3202
- Other reference link(s):

### CODE REVIEW

File: privilege-management.md [hunk 1]
```diff
@@ -306,9 +301,31 @@ Requires the `CREATE USER` privilege.
...
 Requires the `GRANT` privilege with the privileges granted by `GRANT`.
 
+Requires additional `CREATE USER` privilege when create user implicitly.
+
+`GRANT ROLE` requires `SUPER` privilege.
+
 ### REVOKE
 
-Requires the `SUPER` privilege.
+Requires the `GRANT` privilege and those privileges targeted by `REVOKE` statement.
```
Comment (TomShawn) [inline privilege-management.md:310]:
```suggestion
Requires the `GRANT` privilege and those privileges targeted by the `REVOKE` statement.
```

File: privilege-management.md [hunk 2]
```diff
@@ -306,9 +301,31 @@ Requires the `CREATE USER` privilege.
...
+
+`REVOKE ROLE` requires `SUPER` privilege.
+
+### SET GLOBAL
+
+Requires `SUPER` privielges.
```
Comment (TomShawn) [inline privilege-management.md:316]:
```suggestion
Requires `SUPER` privilege to set global variables.
```

File: privilege-management.md [hunk 3]
```diff
@@ -306,9 +301,31 @@ Requires the `CREATE USER` privilege.
...
+
+### ADMIN
+
+Requires `SUPER` privielges.
```
Comment (TomShawn) [inline privilege-management.md:320]:
```suggestion
Requires `SUPER` privilege.
```

File: privilege-management.md [hunk 4]
```diff
@@ -306,9 +301,31 @@ Requires the `CREATE USER` privilege.
...
+
+### SET DEFAULT ROLE
+
+Requires `SUPER` privielges.
```
Comment (TomShawn) [inline privilege-management.md:324]:
```suggestion
Requires `SUPER` privilege.
```

File: privilege-management.md [hunk 5]
```diff
@@ -306,9 +301,31 @@ Requires the `CREATE USER` privilege.
...
+
+### KILL
+
+Requires `SUPER` privielges when killing other user connections.
```
Comment (TomShawn) [inline privilege-management.md:328]:
```suggestion
Requires `SUPER` privilege to kill other user sessions.
```

File: privilege-management.md [hunk 6]
```diff
@@ -286,6 +279,8 @@ Requires the `INSERT` and `SELECT` privileges for the table.
 
 `SHOW CREATE VIEW` requires the `SHOW VIEW` privilege.
 
+`SHOW GRANTS` requires the `SELECT` privilege to the `mysql` database. If the target user is current user, `SHOW GRANTS` dosen't require any privilege.
```
Comment (TomShawn) [inline privilege-management.md:282]:
```suggestion
`SHOW GRANTS` requires the `SELECT` privilege to the `mysql` database. If the target user is current user, `SHOW GRANTS` does not require any privilege.
```

File: privilege-management.md [hunk 7]
```diff
@@ -212,6 +199,8 @@ You can check privileges of TiDB users in the `INFORMATION_SCHEMA.USER_PRIVILEGE
 | Insert         | `InsertPriv`     | Inserts data to a table             |
 | Update         | `UpdatePriv`     | Updates the table data             |
 | Delete         | `DeletePriv`     | Deleted the table data             |
+| Reload         | `ReloadPriv`     | Executes the `FLUSH` statement       |
+| Config         | `ConfigPriv`     | Dynamicly reload configuration             |
```
[the remark this answers is not in the document]
Comment (wwar) [reply privilege-management.md:203]: Dynamically*

[the remark this answers is not in the document]
Comment (TomShawn) [reply privilege-management.md:203]: @wwar Thanks

**Pull request merged**